Tests/Extractor/FrontendExtractorTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Bazinga\JsTranslationBundle\Tests\Extractor;
+
+use Bazinga\Bundle\JsTranslationBundle\Extractor\FrontendExtractor;
+use Bazinga\Bundle\JsTranslationBundle\Tests\WebTestCase;
+use Symfony\Component\Translation\MessageCatalogue;
+
+final class FrontendExtractorTest extends WebTestCase
+{
+    const TEST_LOCALE = 'en';
+    const TEST_KEY_1 = 'test-key-1';
+
+    /**
+     * @var FrontendExtractor
+     */
+    protected $extractor;
+
+    public function setUp()
+    {
+      $container = $this->getContainer();
+      $this->extractor = $container->get('bazinga.jstranslation.translation_frontend_extractor');
```

Comment:
OK, this is the *only* place where you need the service to be public. But there is a much simpler way to handle all this: don't use a WebTestCase to test the FrontendExtractor but instantiate it directly in the test (the only other service it depends on is the Filesystem, which is dead easy to instantiate)

Reply:
OK